types/circular-json/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
+		"dt-header": false,
```

Comment:
Seems like this would be fixed if you just removed the patch version -- that is ignored anyway.